cli.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from api_client import APIClient
+from app.schemas import UserCreate, ConversationCreate, MessageBase, UserUpdate, UserResponse
+from cli_utils import format_conversations
+
+
+class Chat:
+    def __init__(self, user_id: str, base_url: str):
+        self.user = UserCreate(user_id=user_id)
+        self.api_client = APIClient(base_url=base_url)
+        self.conversation = ConversationCreate(user_id=user_id, text="")
+
+    def process(self):
+        self._create_user()
+        while True:
+            user_input = input("You: ")
+            if user_input.lower() == "exit":
+                print("Goodbye!")
+                break
+
+            self.conversation.text += f"User {self.user.user_id}: {user_input}\n"
+
+            if user_input.lower() == 'get conversations':
+                self._handle_query(category='conversations', user_query="Enter a user to search for: ")
+
+            elif user_input.lower() == 'get users':
+                self._handle_query(category='users', user_query="Enter a keyword to search for: ")
+
+            else:
+                completion = self.api_client.send_message(MessageBase(prompt=user_input))
+                self.conversation.text += f"GPT: {completion}\n"
+                print(f"GPT: {completion}")
+
+        self.api_client.create_conversation(conversation=self.conversation)
```

Comment:
If somehow the code will stop running before this point (user will end it with ctrl+c instead of enter 'exit'), this code will not run and the conversation won't be logged. make sure the conversation will always be logged.